### ISSUE TITLE
Unable to disable Optimize taskbar web search results for screen readers

### ISSUE THREAD

**Issue by buddyb99** [da2x/EdgeDeflector#94]
Windows 10 Version 2004 (Build 19041.450)

I am new to this, so it's probably due to the fact that I don't know what I'm doing - although I can follow instructions.  

I am unable to disable Optimize taskbar web search results for screen readers.

I'm stuck at this step "Microsoft Edge settings > view advanced settings > optimize taskbar web search results for screen readers".

My Edge browser only has settings, it does not have advanced settings, and I do not see a toggle to "optimize taskbar web search results for screen readers".

I read an alternative approach to edit the registry. I was to navigate to:

"HKEY_CURRENT_USER\Software\Classes\Local Settings\Software\Microsoft\Windows\CurrentVersion\AppContainer\Storage\microsoft.microsoftedge_8wekyb3d8bbwe\MicrosoftEdge\Main"

and then shift to the right pane, locate and double-click the DWORD 
OptimizeWindowsSearchResultsForScreenReaders. 

But there is no "OptimizeWindowsSearchResultsForScreenReaders" for me to edit.

What am I overlooking?

Thanks,
Buddy

**Comment by da2x** [comment]
Hi, that step only applied to the old version of Edge. I’ve removed this step from the troubleshooting guide, as most users will have been updated away from that version of Edge.

Is it still not working for you? The other troubleshooting steps are more often the problem than this step ever was.

**Comment by buddyb99** [comment]
Sorry for the trouble, Daniel. I panicked a little bit, but your code is working GREAT!  Giving you a sincere virtual high-five!  Thanks again for sharing it with everyone.
